Dockerfile

```diff
@@ -53,16 +53,21 @@ RUN echo "@cloudposse https://apk.cloudposse.com/3.9/vendor" >> /etc/apk/reposit
 # Use TLS for alpine default repos
 RUN sed -i 's|http://dl-cdn.alpinelinux.org|https://alpine.global.ssl.fastly.net|g' /etc/apk/repositories && \
     echo "@testing https://alpine.global.ssl.fastly.net/alpine/edge/testing" >> /etc/apk/repositories && \
-    echo "@community https://alpine.global.ssl.fastly.net/alpine/edge/community" >> /etc/apk/repositories && \
-    apk update
+    echo "@community https://alpine.global.ssl.fastly.net/alpine/edge/community" >> /etc/apk/repositories
 
 # Install alpine package manifest
 COPY packages.txt /etc/apk/
+# Install repo checksum in an attempt to ensure updates bust the Docker build cache
+COPY geodesic_apkindex.md5 /var/cache/apk/
+COPY rootfs/usr/local/bin/geodesic-apkindex-md5 /tmp/
```

Comment:
Why `/tmp`? I think we can keep it.

Reply:
Because active development if Geodesic mainly involves making changes to rootfs, and the full build of Geodesic takes over 10 minutes, I want to keep the copy of rootfs as the last step. So we temporarily install the script in /tmp so we can run it now. We later (at the end of the Dockerfile) install it permanently when we copy over the entire rootfs.

Reply:
I guess my point was why not just `COPY rootfs/usr/local/bin/geodesic-apkindex-md5 /usr/local/bin`; I understand why not to copy everything.